pkg/template/functions.go

```diff
@@ -0,0 +1,35 @@
+package template
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+)
+
+// FuncMap defines the available functions to kubermatic templates.
+var funcs = template.FuncMap{
```

Comment:
This can be inlined into `TxtFuncMap` again?

Reply:
good point